doc/faq.md

```diff
@@ -57,6 +57,65 @@ directory. None of this should affect any existing Haskell tools at all.
   there may be more options here for new projects in the future (see issue
   [253](https://github.com/commercialhaskell/stack/issues/253))
 
+### Cabal-the-library
+
+The relationship between stack and Cabal-the-library is complicated.
+
+1. Stack itself builds against a version of the Cabal library, which it uses for parsing Cabal files.
```

Comment:
Worth pointing out that this determines which version of the cabal file format Stack is able to parse.

Reply:
ditto